docs/firefly/verify_download.md

```diff
@@ -1,20 +1,35 @@
 ---
-description: Verify your Firefly download on Windows, maxOS and/or Linux.
+description: Firefly is IOTA's new official wallet for Chrysalis. You can verify your Firefly download on Windows, maxOS, and/or Linux.
 image: /img/logo/Chrysalis_logo_dark.png
 keywords:
 - Firefly
+- DLT
+- wallet
 - download
 - verify
 - windows
 - macOS
 - linux
+- how to
 ---
+# Firefly
+
+:::info
+
+Firefly is IOTA's new official wallet for Chrysalis. 
+
+:::
+
+**Website**: [https://firefly.iota.org/](https://firefly.iota.org/)
+
+**Code Repository**: [https://github.com/iotaledger/firefly](https://github.com/iotaledger/firefly)
+
 # Verify your Firefly Desktop Download
```

Comment:
Docusuarus only accepts one h1, please  make this a h2 and add one level to subsequent headers

Reply:
👌